docs/howto_config.md

```diff
@@ -50,6 +50,38 @@ nodeName: ""
 logVLevel: 0
 ```
 
+## Service NodePort range
+
+The `serviceNodePortRange` setting allows the extension of the port range available
+to NodePort Services. This option is useful when specific standard ports under the
+`30000-32767` need to be exposed. i.e., your device needs to expose the `1883/tcp`
+MQTT port on the network because some client devices cannot use a different port.
+
+If you use this option, you must be careful; NodePorts can overlap with system ports,
+causing malfunction of the system or MicroShift. Take the following considerations
+into account:
+
+* Do not create any NodePort service without an explicit `nodePort` selection. The port
+  will be assigned randomly by the kube-apiserver.
+* Do not create any NodePort service for any system service port, MicroShift port,
+  or other services you expose on your device HostNetwork.
+
+List of ports that you must avoid:
+
+| Port          | Description                                                     |
+|---------------|-----------------------------------------------------------------|
+| 22/tcp        | SSH port
+| 1936/tcp      | Metrics service for the openshift-router, not exposed today
```

Comment:
add 80 and 443 ports exposed for route deployment?

Reply:
true! thanks I was looking at the netstat -nltp list which does not cover that.

Reply:
Done